gunicorn/workers/gaiohttp.py

```diff
@@ -5,13 +5,23 @@
 
 import sys
 
-if sys.version_info >= (3, 3):
+from gunicorn import util
+
+if sys.version_info >= (3, 4):
     try:
         import aiohttp  # NOQA
     except ImportError:
         raise RuntimeError("You need aiohttp installed to use this worker.")
     else:
-        from gunicorn.workers._gaiohttp import AiohttpWorker
+        try:
+            from aiohttp.worker import GunicornWebWorker as AiohttpWorker
+        except ImportError:
+            from gunicorn.workers._gaiohttp import AiohttpWorker
+
+        util.warn(
+            "The 'gaiohttp' worker is deprecated. Please install 'aiohttp' "
```

Comment:
i would would reformulate by saying that  "The 'gaiohttp' worker is deprecated. Please look at "link to worker page" for an alternative worker supporting asyncio.